source/_docs/configuration/templating.markdown

```diff
@@ -278,6 +278,25 @@ The same thing can also be expressed as a filter:
 
 {% endraw %}
 
+### Integrations
+
+- `integration_entities(integration)` returns a list of entities that are associated with a given integration, such as `hue` or `zwave_js`.
+- `integration_entities(title)` if you have multiple instances set-up for an integration, you can also use the title you've set for the integration in Settings --> Integrations. For example to only target a specific device bridge.
```

Comment:
```suggestion
- `integration_entities(title)` if you have multiple instances set-up for an integration, you can also use the title you've set for the integration in case you only want to target a specific device bridge.
```